ush/generate_FV3LAM_wflow.py

```diff
@@ -213,8 +213,11 @@ def generate_FV3LAM_wflow(ushdir, logfile: str = "log.generate_FV3LAM_wflow", de
         # from the jinja template file.
         #
         try:
+            # Until all of RRFS is merged lets temporarily ignore undefined template variables
+            # This is needed to have all the workflow in at the beginning, then add testable tasks one
+            # one by one. Once everything is merged this should be removed.
```

Comment:
This can be removed right before this PR is merged so that the strict variable checking in fill_jinja is maintained. But for testing you would need this.